ffig/generators/go.py

```diff
@@ -4,10 +4,39 @@
 # for generator plugins. Ultimately, it will generate Go packages in the
 # correct structure.
 
+import os
+import os.path
+
 import generators
+import logging
```

Comment:
Already used it. I started to add 'print' statements in `generators/__init__.py` while debugging the import magic, and wanted to keep the info available in case anyone developing generators needs it.

It's pretty useful, I think. It just logs to stderr, so it's not going to leave logfiles or anything; just useful for debugging, or anyone who wants to get an idea of how it works.